src/Command/CleanupCommand.php

```diff
@@ -13,13 +13,26 @@
 
 namespace Sonata\NotificationBundle\Command;
 
+use Sonata\NotificationBundle\Backend\BackendInterface;
 use Sonata\NotificationBundle\Backend\QueueDispatcherInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class CleanupCommand extends ContainerAwareCommand
+class CleanupCommand extends Command
```

Comment:
This is Bc break

Reply:
This is one BC break we already accepted on other projects. But Im not sure that's a good argument.

Since we're trying to release next sonata major, we can report this PR to master indeed

Reply:
IMO given we decided to not support symfony 5 on current version, we should move this to master. Previously we did some BC breaks when we were not sure if we could make it fully compat with sf5

Reply:
You're right

Reply:
No problem but i'm pretty sure I need to cherry pick and open another PR